packages/flame_yarn/yarn/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: yarn
```

Comment:
See discussion in #contributors.

Reply:
I don't think this have to do with our discussion right? @spydon simply is suggesting to rename the package to avoid confusion with node's package manager also called yarn

Reply:
There was a thread for the name too 😅 
I think that if the yarn spinner people agree that we name it yarn_spinner on pub.dev I think that's the best to roll with. :)